ete4/smartview/gui/static/js/draw.js

```diff
@@ -324,7 +324,7 @@ async function draw_aligned(params, npanels) {
                  h: Math.max(div_tree.offsetHeight / zy + view.tl.y - view.tree_size.height,
                         view.aligned.footer.height / zy) } }
     };
-
+    
```

Comment:
The trailing whitespaces can (rarely) be a source of bugs very difficult to track. It also pollutes the diffs.

If you use vscode you can for example avoid it going to settings and activating the option to trim trailing whitespace on saving. See https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut